apps/web/src/components/(dashboard)/avatar/stack-avatars-ui.tsx

```diff
@@ -0,0 +1,52 @@
+'use client';
+
+import { useWindowSize } from '@documenso/lib/client-only/hooks/use-window-size';
+import type { Recipient } from '@documenso/prisma/client';
+import { Popover, PopoverContent, PopoverTrigger } from '@documenso/ui/primitives/popover';
+import {
+  Tooltip,
+  TooltipContent,
+  TooltipProvider,
+  TooltipTrigger,
+} from '@documenso/ui/primitives/tooltip';
+
+import { StackAvatars } from './stack-avatars';
+import { StackAvatarsComponent } from './stack-avatars-component';
+
+export type StackAvatarsUIProps = {
+  recipients: Recipient[];
+  position?: 'top' | 'bottom';
+  children?: React.ReactNode;
+};
+
+export const StackAvatarsUI = ({ recipients, position, children }: StackAvatarsUIProps) => {
+  const size = useWindowSize();
```

Comment:
As per my comment, I'd avoid this and simply use the popover component at all times with a `onMouseOver` handler which can deal with adding the old hover behaviour back.

Reply:
```
  <Popover open={isOpen}>
     <PopoverTrigger
       className="flex cursor-pointer"
       onMouseOver={() => setIsOpen(true)}
       onMouseLeave={() => setIsOpen(false)}
     >
      {children || <StackAvatars recipients={recipients} />}
  </PopoverTrigger>
```

If we add `onMouseOver` and `onMouseLeave`  on Popover to give the tooltip vibe then the copy recipients email feature won't work.